versions/1/elasticsearch/spec.yml

```diff
@@ -0,0 +1,28 @@
+spec:
```

Comment:
You need to add reference to this file in the package's spec: https://github.com/elastic/package-spec/blob/master/versions/1/spec.yml